dandiapi/api/views/dandiset.py

```diff
@@ -110,15 +110,20 @@ def create(self, request):
         # Strip away any computed fields
         metadata = Version.strip_metadata(metadata)
 
-        metadata['contributor'] = [
-            {
-                'name': f'{request.user.first_name} {request.user.last_name}',
-                'email': request.user.email,
-                'roleName': ['dandi:ContactPerson'],
-                'schemaKey': 'Person',
-                'includeInCitation': True,
-            }
-        ]
+        # Only inject a default contributor field if it is not specified in the metadata
+        metadata = {
+            'contributor': [
+                {
+                    'name': f'{request.user.first_name} {request.user.last_name}',
```

Comment:
Dandischema requires personal names to be in the form "familyname, given names".

Reply:
@dchiquito - i don't know if this comment from @jwodder got lost, but this will not pass validation.

Reply:
This test does not actually validate the metadata, but I will update this for consistencies sake in another PR.

Reply:
Thank you for pointing this out, I will fix it in another PR.